src/app/sidenav/sidenav.component.html

```diff
@@ -0,0 +1,8 @@
+<mat-sidenav-container fullscreen [style.top.px]="screenWidth <= 599 ? 56 : 64">
```

Comment:
Hack to get proper distance from the top of the screen and also have the `fullscreen` attribute.